run.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# This scripts runs runtime ntrip docker container and
+#  passes any arguments to the ntrip_client app
```

Comment:
"ntrip_client app" App? Do you mean the python script?

Perhaps a bit confusing use of "app" since we actually _do_ have and use ntrip client apps on our phones, so I'd recommend calling it Python script or code

Reply:
`run.sh` has been updated and renamed to `runtime.sh`. This new version contains a different comment